contracts/ARC20.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0;
+
+library NativeAssets {
+    address constant balanceAddr = 0x0100000000000000000000000000000000000001;
+    address constant transferAddr = 0x0100000000000000000000000000000000000002;
+
+    function assetBalance(address addr, uint256 assetID) public returns (uint256) {
```

Comment:
Shouldn't this be a `view`?

Reply:
FWWI I compiled the code and didn't get any warnings/errors.

Reply:
Ya it should be, will require changing `call` to `staticcall`.